### ISSUE TITLE
feat(datamodel): domain name field in Organisation Identity

### ISSUE THREAD

**Issue by sudesh0sudesh** [LuatixHQ/opencti#5912]
## Use case

Can we have a domain name field in organisation Identity contact information

## Current Workaround

Currently there is Contact information

## Proposed Solution

We think it would be great to have domain Name in Organisation identity, even as sub field in contact. Which can be used for enrichment or using it for dns_twist for tracking domain squatting

**Comment by sudesh0sudesh** [comment]
<img width="829" alt="Screenshot 2024-02-10 at 17 28 00" src="https://github.com/OpenCTI-Platform/opencti/assets/11227791/a8656cb2-5b55-4a3a-bcbc-1288fe3e73d1">

**Comment by nino-filigran** [comment]
@sudesh0sudesh As you may know since we already have the observable domain, we would rather encourage you to use a relation mechanism between your org and the domain name. Isn't this sufficient for you?

**Comment by sudesh0sudesh** [comment]
Hello @nino-filigran 

If someone wants to run dnstwist against a specific domain, it creates a relationship but to distinguish from the main domain we have to apply labels and in future, if someone wants to run dnstwist against the domain then they have to go to that specific domain and execute it. Not only that. If someone wants to create a connector that checks specific things against the organisation domain. Rather than running against the domain. They can run on the connector against the organisation object.

**Comment by sudesh0sudesh** [comment]
Maybe a niche case

**Comment by nino-filigran** [comment]
Yeah I understand your issue. 
And in this context, what do you think about having a different type of relation (custom, not stix), like "offcially owns" (to be able to have a relation that represents the fact that your organisation owns some domains, while using the related to link the malicious ones)?

The reason why I'm trying to use relations is to try to remain coherent with the platform and void having duplication of data/difficulty of maintenance.

**Comment by sudesh0sudesh** [comment]
yeah may be new relation should work, It would also tackles if organisation has multiple domains they can all be linked
